CMakeLists.txt

```diff
@@ -124,6 +124,17 @@ if(CAF_ENABLE_TESTING)
       endif()
     endforeach()
   endfunction()
+  function(caf_add_legacy_test_suites target)
```

Comment:
You can actually remove `caf_add_test_suites` now. We won't use it anymore. Same for the `TEST_SOURCES` and `TEST_SUITES` markers.